client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -27,6 +27,12 @@ import Gravatar from 'components/gravatar';
 import i18n from 'lib/mixins/i18n';
 
 /**
+ * Constants
+ */
+
+const STATS_PAGE = '/stats/insights/';
```

Comment:
Nah, seems ok. The problem is the default view changes, so it'd be nice to not force it here.